qa/L0_input_validation/input_validation_test.py

```diff
@@ -208,7 +208,7 @@ def get_input_array(input_size, np_dtype):
             triton_client.infer(model_name=model_name, inputs=inputs)
         err_str = str(e.exception)
         self.assertIn(
-            f"expected {input_size} string elements for inference input 'INPUT1', got {input_size+2}",
+            f"unexpected number of string elements {input_size+1} for inference input 'INPUT1' for model '{model_name}', expecting {input_size}",
```

Comment:
Should this be `+1` or `+2`? Old test was `+2`, and I see `inputs = get_input_array(input_size + 2, np_dtype_string)`

Reply:
+1 because I added early termination to the core check for string input tensor. An error will be returned immediately when the code iterates the 17th string from a 16-string input.

Reply:
Also, the error message is consistent with the GPU string input size check at backend.
https://github.com/triton-inference-server/backend/blob/30fa78aabe827e071c353dec3d4eabbcc115573a/src/backend_common.cc#L1389-L1398